package.json

```diff
@@ -18,7 +19,7 @@
 		"url": "https://github.com/gep13-oss/clipimg-vscode"
 	},
 	"engines": {
-		"vscode": "^1.31.0"
+		"vscode": "^1.48.0"
```

Comment:
Just FYI, this is a breaking change.
This will drop any ability to install/use clipimg on vscode versions less than 1.48.0 (basically will be equal to `>=1.48.0 && < 2.0.0`)

Reply:
Yes, probably fine by the time this addin is beyond hello world and released. 😎

Reply:
oh right. I somehow mixed it up with a different vscode extension that actually have an existing release 🤦‍♂️